.github/workflows/job-postcommit-java-validaterunners-samza.yml

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# To learn more about GitHub Actions in Apache Beam check the CI.md
+
+name: Java VR Samza PostCommit
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 */6 * * *'
+  push:
+    branches: ['master', 'release-*']
+    tags: ['v*']
+permissions: read-all
+
+jobs:
+  java_vr_samza:
+    name: Run Samza Java ValidatesRunner
```

Comment:
i think it would be good to add the Samza ValidatesRunner sickbay workflow to run as well

Reply:
I don't think we should add it to the same workflow since we should keep it separately in case this last test fails (since I think it fails/passes several times), it will cause the whole workflow fails... 
What do you think @damccorm ?

Reply:
Yeah, I agree - I think having a separate sickbay workflow makes more sense since adding it here will affect the whole workflow's status